tests/unit/analytics/test_result_analysis.py

```
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pyrit.analytics.result_analysis import AttackStats, analyze_results
+from pyrit.models import AttackOutcome, AttackResult
+
+
+# Helper to create mock AttackResult with just enough fields
+def make_attack(outcome: AttackOutcome, attack_type: str = "default") -> AttackResult:
+    return AttackResult(
+        conversation_id="1", objective="test objective", attack_identifier={"type": attack_type}, outcome=outcome
+    )
+
+
+def test_analyze_results_empty():
+    result = analyze_results([])
```

This should probably raise an exception because it makes no sense